docs/getting-started/1201-ci-environment.md

```diff
@@ -34,6 +35,54 @@ values={[
 
 </TabItem>
 
+<TabItem value="travisci">
+
+```yaml
+---
+os: linux
+arch: amd64
+dist: bionic
+language: minimal
+
+env:
+  global:
+    - DAGGER_VERSION: 0.2.19
```

Comment:
this variable is not being used anywhere?

Reply:
Yes, because is defined at the worker level, it acts like a regular environment variable